[2024-01-31T21:24:40.505+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: big_query_dashboard_final.create_dataset manual__2024-01-31T21:24:20.051046+00:00 [queued]>
[2024-01-31T21:24:40.526+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: big_query_dashboard_final.create_dataset manual__2024-01-31T21:24:20.051046+00:00 [queued]>
[2024-01-31T21:24:40.529+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-31T21:24:40.565+0000] {taskinstance.py:2191} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2024-01-31 21:24:20.051046+00:00
[2024-01-31T21:24:40.578+0000] {standard_task_runner.py:60} INFO - Started process 376 to run task
[2024-01-31T21:24:40.584+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'big_query_dashboard_final', 'create_dataset', 'manual__2024-01-31T21:24:20.051046+00:00', '--job-id', '525', '--raw', '--subdir', 'DAGS_FOLDER/Main_DAGS.py', '--cfg-path', '/tmp/tmpt3el0s3i']
[2024-01-31T21:24:40.595+0000] {standard_task_runner.py:88} INFO - Job 525: Subtask create_dataset
[2024-01-31T21:24:40.682+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-01-31T21:24:40.975+0000] {task_command.py:423} INFO - Running <TaskInstance: big_query_dashboard_final.create_dataset manual__2024-01-31T21:24:20.051046+00:00 [running]> on host e7e4b550b444
[2024-01-31T21:24:41.627+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='big_query_dashboard_final' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T21:24:20.051046+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T21:24:20.051046+00:00'
[2024-01-31T21:24:41.649+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-01-31T21:24:41.660+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-31T21:24:41.673+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-31T21:24:42.008+0000] {bigquery.py:469} INFO - datasetId was not specified in `dataset_reference`. Will use default value dashboard_dataset.
[2024-01-31T21:24:42.050+0000] {bigquery.py:469} INFO - projectId was not specified in `dataset_reference`. Will use default value wine-de-dashboard.
[2024-01-31T21:24:42.090+0000] {bigquery.py:480} INFO - Creating dataset: dashboard_dataset in project: wine-de-dashboard 
[2024-01-31T21:24:43.113+0000] {bigquery.py:2040} INFO - Dataset dashboard_dataset already exists.
[2024-01-31T21:24:43.375+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=big_query_dashboard_final, task_id=create_dataset, execution_date=20240131T212420, start_date=20240131T212440, end_date=20240131T212443
[2024-01-31T21:24:43.534+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T21:24:43.684+0000] {taskinstance.py:3280} INFO - 5 downstream tasks scheduled from follow-on schedule check
